### ISSUE TITLE
I can't use certain formatting in prompts!

### ISSUE THREAD

**Issue by Ascendant-Stoic** [JonathanFly/bark#55]
When I attempt to use the symbol "♪" in the prompt to indicate a singing voice it just gives me an error?, do I need to format it in a specific way (besides this -> ♪ I'm the king of the jungle ♪ ) or is there a setting that I need to check first (typing [singing] seems to sometimes work making the voice sing what comes after it, is that intentional or what?)

Also the using the speaker like in (Man: Hi ... Woman: Hello) isn't consistent at all, is there a setting I need to adjust for it to work?

**Comment by JonathanFly** [comment]
Yup, this a bug. Set the output filename to something specific and it will work for now.
